libs/utils/energy.py

```diff
@@ -325,4 +343,133 @@ def report(self, out_dir, out_energy='energy.json', out_samples='samples.csv'):
 
         return (channels_nrg, nrg_file)
 
+
+class IIOCapture(EnergyMeter):
+    "iio-capture Energy Meter for BayLibre ACME board"
+
+    def __init__(self, target, conf, res_dir):
+        super(IIOCapture, self).__init__(target)
+
+        self._iiocapturebin = conf['iiocapturebin']
```

Comment:
Can we make this an optional configuration parameter, so that the user is not forced to express it in the configuration file:
```python
self._iiocapturebin = DEFAULT_ENERGY_METER['acme']['iiocapturebin']
#...
try:
    self._iiocapturebin = conf['iiocapturebin']
except: pass
```

The same should be done for all other configuration properties.